### PR TITLE
web: add the shortcut dialog to the sidebar

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu, @maiamcc,

Please review the following commits I made in branch nicks/ch9620/dialog:

f84be98c693dc1691acf63f194d34c6d1eeeec17 (2020-10-14 16:17:00 -0400)
web: add the shortcut dialog to the sidebar

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
Screenshot from 2020-10-14 16-17-28

i also tried to reconcile the differences between the two dialogs

File: web/src/SidebarAccount.tsx [hunk 1]
```diff
@@ -252,30 +186,125 @@ function SidebarAccount(props: SidebarAccountProps) {
...
+      e.preventDefault()
+    }
+  }
+
+  render() {
+    return <span></span>
+  }
+}
+
+function SidebarAccount(props: SidebarAccountProps) {
```
Comment (maiamcc) [inline web/src/SidebarAccount.tsx:227]: nbd but might consider renaming "SidebarAccount" as it's no longer just account info. "SidebarInfo"?

File: web/src/SidebarAccount.tsx [hunk 2]
```diff
@@ -252,30 +186,125 @@ function SidebarAccount(props: SidebarAccountProps) {
...
+  const [shortcutsDialogOpen, setShortcutsDialogOpen] = useState(false)
+  const [accountMenuOpen, setAccountMenuOpen] = useState(false)
+
+  let toggleAccountMenu = () => setAccountMenuOpen(!accountMenuOpen)
+  let toggleShortcutsDialog = () => setShortcutsDialogOpen(!shortcutsDialogOpen)
+
+  let optionalLearnMore = null
```
Comment (maiamcc) [inline web/src/SidebarAccount.tsx:234]: maybe optionalTiltCloudLearnMore / optionalTCLearnMore? (Again, FFTI)

**Review by maiamcc — APPROVED** [review]
nice!

File: web/src/SidebarAccount.tsx (hunk 1 above)
Comment (nicks) [reply web/src/SidebarAccount.tsx:227]: the diff is misleading - this isn't new code, this was all here before. i'd be open to a PR to rename things a bit, though not sure we should get too hung up on names right now if this is all changing anyway.

File: web/src/SidebarAccount.tsx (hunk 2 above)
Comment (nicks) [reply web/src/SidebarAccount.tsx:234]: ditto

**Pull request merged**